domains/leon.json

```diff
@@ -1,11 +1,10 @@
 {
-  "repo": "https://github.com/LeonskiDev/leonskidev.github.io",
-  "description": "This is my portfolio website, if possible I'll also have bodged.is-a.dev as my blog.",
+  "repo": "https://github.com/LeonskiDev/leonskidev",
   "owner": {
     "username": "LeonskiDev",
     "email": "bodged@pm.me"
   },
   "record": {
-    "CNAME": "leonskidev.github.io"
+    "CNAME": "leonskidev.github.io/leonskidev/"
```

Comment:
CNAME has to be a host name. If you're having trouble setting up your domain, [open up a discussion](https://github.com/is-a-dev/register/discussions) or create a support issue

Reply:
Oh damn forgot about that, whoops. Thanks for reminding me.